skpro/distributions/beta.py

```diff
@@ -0,0 +1,65 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Beta probability distribution."""
+
+__author__ = ["malikrafsan"]
+
+import pandas as pd
+from scipy.stats import beta, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class Beta(_ScipyAdapter):
+    r"""Beta distribution.
+
```

Comment:
could you add a short note about parameterization? Have a look at the other distributions.

The reason is, many distributions have different parameterizations, e.g., normal can be parameterized by sigma or sigma-squared. It suffices to spell out one of the distribution defining functions, usually I choose the "simplest" for users to understand.

Let me know if you are unsure, then I will add it later.

Reply:
Recently, I have added the parameterization note about the beta distribution. I had a look at the other distributions, and a lot of them are noted by PDF or CDF formula. That's why I tried to add the PDF formula as I think it is easier to understand, in the context of Beta Distribution. I took the formula from the [Wikipedia](https://en.wikipedia.org/wiki/Beta_distribution). I added the PDF formula as given

$$f(x) = \frac{x^{\alpha-1}(1-x)^{\beta-1}}{B(\alpha, \beta)}$$

Please let me know if there is something I missed! Also, I am totally fine as well if you want to modify the PR as well

Reply:
yes, that works, especially with the explanation that B is for normalization.